packages/@react-spectrum/s2/src/IllustratedMessage.tsx

```diff
@@ -114,9 +114,7 @@ const illustration = style<IllustratedMessageStyleProps & {isInDropZone?: boolea
   '--iconPrimary': {
     type: 'color',
     value: {
-      // TODO: ask design about what the color should be. Says gray-800 in the designs file, neutral in token spec, but different neutral in dropzone spec
-      default: 'gray-800',
-      isInDropZone: 'gray-500', // neutral doesn't seem to match the color in designs, opted for gray-500 instead
+      default: 'neutral',
```

Comment:
according to design, the illustration should have the same color both in an IllustratedMessage and in a DropZone. the design files do not currently reflect this but there are tickets to update them